tests/test_mint.cairo

```diff
@@ -160,3 +162,83 @@ fn test_is_public_buy() {
     assert(tokenized_cc.len() == 20, 'cc should have 20 element');
 }
 
+// get_available_money_amount
+
+#[test]
+fn test_get_available_money_amount() {
+    let owner_address: ContractAddress = contract_address_const::<'OWNER'>();
+    let (project_address, _) = deploy_project();
+    let (erc20_address, _) = deploy_erc20();
+    let (minter_address, _) = deploy_minter(project_address, erc20_address);
+
+    let times: Span<u64> = get_mock_times();
+    let absorptions: Span<u64> = get_mock_absorptions();
+    // Setup the project with initial values
+    setup_project(project_address, 8000000000, times, absorptions,);
+
+    // Start testing environment setup
+    start_prank(CheatTarget::One(minter_address), owner_address);
+    start_prank(CheatTarget::One(erc20_address), owner_address);
+    start_prank(CheatTarget::One(minter_address), owner_address);
+    start_prank(CheatTarget::One(erc20_address), owner_address);
+
+    let project = IAbsorberDispatcher { contract_address: project_address };
+    assert(project.is_setup(), 'Error during setup');
+
+    let minter = IMintDispatcher { contract_address: minter_address };
+
+    // Ensure the public sale is open
+    let sale_open = minter.is_public_sale_open();
+    assert(sale_open == true, 'public sale not open');
+
+    // Default initial amount of money = 8000000000
+    let initial_money: u256 = 8000000000;
+
+    // Verify if the initial value is correct
+    let remaining_money = minter.get_available_money_amount();
+    assert(remaining_money == initial_money, 'remaning money wrong value');
```

Comment:
typo 'remaining' instead of 'remaning'. Also edit the other same typo on that file